tools/upgrade-version/src/main.rs

```diff
@@ -90,6 +90,11 @@ fn main() -> anyhow::Result<()> {
                 .action(clap::ArgAction::SetTrue)
                 .help("Also bump versions in C# SDK and templates"),
         )
+        .group(
+            ArgGroup::new("update-targets")
+                .args(["typescript", "rust-and-cli", "csharp"])
+                .required(true),
+        )
```

Comment:
this is unrelated, but I noticed that it was possible to run `cargo bump-versions 1.5.0` without any args, which is a no-op since https://github.com/clockworklabs/SpacetimeDB/pull/3308.